Components/badges.css

```diff
@@ -0,0 +1,92 @@
+.badge-container {
+  position: relative;
+  cursor: pointer;
+}
+
+.badge-red {
+  background-color: var(--dark-danger-color);
```

Comment:
No need to write --colorName-color because it's obvious as color. Just --colorName is enough.